src/data/equiv/list.lean

```diff
@@ -245,6 +245,13 @@ end denumerable
 
 namespace equiv
 
+/-- The type of natural numbers is naturally equivalent to lists on unit. -/
```

Comment:
Nitpick: "canonically" is perhaps better than "naturally" here. :-) Maybe also reverse the order of the comment.